snippets/json-schema.code-snippets

```diff
@@ -1,118 +1,118 @@
-        "description": "An array of booleans"
+    "Snowboard: Array of arrays property": {
+        "prefix": ["array of arrays"],
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"array\",", "\t\"description\": \"${2:Description goes here}\",", "\t\"items\": {", "\t\t\"type\": \"array\"", "\t}", "}${0}"],
+        "description": "An array of arrays"
     },
-    "Object array property": { // TODO: add the object properties to this
+    "Snowboard: Object array property": {
         "prefix": ["object array"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"object\"", "\t}", "}"],
-        "description": "An array of booleans"
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"object\",", "\t\t\"description\": \"${2:Description goes here}\",", "\t\t\"properties\": {", "\t\t\t${3}", "\t\t},", "\t\t\"additionalProperties\": ${4:false}", "\t}", "}${0}"],
+        "description": "An array of objects"
     },
 
 
     // nullable types, add a 'n' afterwards for shortcuts?
-	"Number property (nullable)": {
-        "prefix": ["number"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": [\"number\", \"null\"],", "}"],
-        "description": "A for loop."
-    },
-    "String property (nullable)": {
-        "prefix": ["string"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": [\"string\", \"null\"],", "}"],
-        "description": "A for loop."
-    },
-    "Integer property (nullable)": {
-        "prefix": ["string"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": [\"integer\", \"null\"],", "}"],
-        "description": "A for loop."
-    },
-    "Boolean property (nullable)": {
-        "prefix": ["boolean"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": [\"boolean\", \"null\"],", "}"],
-        "description": "A for loop."
-    },
-    "Array property (nullable)": {
-        "prefix": ["array"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": [\"array\", \"null\"],", "\t\"items\": {", "\t\t\"type\": \"${2:type}\"", "\t}", "}"],
-        "description": "An (untyped) array"
-    },
-    "Object property (nullable)": {
-        "prefix": ["object"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": [\"object\", \"null\"],", "\t\"properties\": {", "\t\t${2}", "\t}", "}"],
-        "description": "An object"
-    },
-    "Datetime / timestamp property (nullable)": {
-        "prefix": ["date-time"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": [\"string\", \"null\"],", "\t\"format\": \"date-time\"", "}"],
-        "description": "A datetime (string) property"
-    },
-    "URI property (nullable)": {
-        "prefix": ["uri"],
-        "body": ["\"${1:name}\": {", "\t\"type\": [\"string\", \"null\"],", "\t\"format\": \"uri\"", "}"],
-        "description": "A uri property"
+	"Snowboard: Number property (nullable)": {
+        "prefix": ["number", "nullable number"],
```

Comment:
If we do short versions, maybe just 'n' + type, so `nstr`, `nbool`, `nint`, etc?

Reply:
The autocomplete search already picks these up correctly.